ui/src/app/components/runtime-configuration-panel.tsx

```diff
@@ -514,29 +389,33 @@ const PanelMain = fp.flow(
                           () => RuntimeStatusRequest.DeleteRuntime,
                         ],
                         [
-                          deletePDSelected && !usingDataproc,
+                          deletePDSelected &&
+                            analysisConfig.computeType !== ComputeType.Dataproc,
                           () => RuntimeStatusRequest.DeleteRuntimeAndPD,
                         ],
                         [
                           // TODO: this configuration is not supported.  Remove?
```

Comment:
Looking at the commen: TODO: this configuration is not supported. Remove? Do we still need this configuration? or will that be part of another PR

Reply:
I don't want to make any functional changes in this PR.  That's a good candidate for near-future work.